src/Features/Auth/actions.js

```diff
@@ -1,8 +1,10 @@
-import { LOGIN, LOGIN_SUCCESS, LOGOUT, LOGIN_ERROR } from '../../scripts/constants';
+import { LOGIN, LOGIN_SUCCESS, LOGOUT, LOGIN_ERROR, SIGNUP } from '../../scripts/constants';
```

Comment:
It did not seem fit to place constants in scripts directory .. please put your ActionTypes in redux folder